src/config-schema.coffee

```diff
@@ -155,32 +172,36 @@ module.exports =
       softTabs:
         type: 'boolean'
         default: true
+        description: 'If the `Tab Type` config setting is set to "auto" and autodetection of tab type from buffer content fails, then this config setting determines whether a soft tab or a hard tab will be inserted when the Tab key is pressed.'
```

Comment:
Great -- thanks for explaining that flow, @benogle. :bow: I just pushed a clarification for how auto-detection works, and going to :boat: now.